scripts/javascript/utils.js

```diff
@@ -48,3 +52,17 @@ function authSocket() {
     });
 }
 exports.authSocket = authSocket;
+function sleep(ms) {
+    return new Promise(resolve => setTimeout(resolve, ms));
+}
+exports.sleep = sleep;
+function fadeOut(el, ms) {
```

Comment:
Hardcoded time handling could create a problem in future

Reply:
Thanks for the heads up, fixed